src/openforms/formio/typing.py

```diff
@@ -5,7 +5,9 @@
 (subsets) of these schema's to make it easier to reason about the code operating on
 (parts of) the schema.
 """
-from typing import TypedDict
+from typing import Any, Optional, TypedDict
```

Comment:
Any and Optional are unused I think

Reply:
my new linter doesn't make it that obvious anymore :<